mbed_host_tests/host_tests_conn_proxy/conn_primitive_serial.py

```diff
@@ -84,6 +87,9 @@ def reset_dev_via_serial(self, delay=1):
 
     def read(self, count):
         """! Read data from serial port RX buffer """
+        # TIMEOUT: Since read is called in a loop, wait for self.timeout period before calling serial.read(). See
+        # comment on serial.Serial() call above about timeout.
+        sleep(self.timeout)
```

Comment:
Exactly since the timer tests we have are for 1 sec granularity they will not be impacted. For anything less then 1 sec host test based verification is not perfect anyways.